tests/connect_redirect/connect_redirect_tests.cpp

```diff
@@ -467,26 +442,23 @@ get_client_socket(bool dual_stack, _Inout_ client_socket_t** sender_socket)
 }
 
 void
-authorize_test_wrapper(_In_ const struct bpf_object* object, bool dual_stack, _In_ sockaddr_storage& destination)
+authorize_test_wrapper(bool dual_stack, _In_ sockaddr_storage& destination)
```

Comment:
```suggestion
authorize_test_wrapper(bool dual_stack, _Inout_ sockaddr_storage& destination)
```
since authorize_test marks destination as Inout